internal/graphql/resolver/operator.go

```diff
@@ -80,10 +77,9 @@ func (r *Resolver) changeOperatorStatus(ctx context.Context, provider models.Pro
 		err = model.RunMeshSync(kubeclient, del)
 		if err != nil {
 			r.Log.Error(err)
-			// r.operatorSyncChannel <- false
 			r.Broadcast.Submit(broadcast.BroadcastMessage{
 				Type:    broadcast.OperatorSyncChannel,
-				Message: false,
+				Message: err,
```

Comment:
I saw that this Message field can be a boolean or an error. This will pose a problem in the future if we cast the variabel just like what we are doing, for example boolean/error but it turns out to be the other way around. Could we just define a new field just in case, like ok: boolean (and is always there true/false), then error is a null / error. What do you think? Hence, if we find ok = false -> we get the error field.

Reply:
this would also make it clearer if the response is bad or not in the switch case in your other file

Reply:
I kept message as `interface{}` to keep it abstract, since it might be vary from different type of the broadcast. Not sure if it would make sense for other Broadcast type. Said that, I also dont like the idea of not knowing what the message type is. 
One way might be to define individual stuct for each type?

Reply:
Although I dont want the overhead of defining a struct type in meshkit once again, in case we are using it to send data locally defined in meshery.

I think adding comments when defining broadcast type about what to expect in message should do for now.